pynicotine/__init__.py

```diff
@@ -37,7 +40,7 @@ def check_arguments():
     )
     parser.add_argument(
         "-u", "--user-data", metavar=_("dir"),
-        help=_("use non-default user data directory for e.g. list of downloads")
```

Comment:
If we remove the -p argument, would it be worth mentioning plugins as an example here?

Reply:
Yes good idea, possibly, the string I have at the moment is:

"run with alternative user profile directory"

Ultimately, the string will be used on `--profile`, but I am tentative to implement `-p` straight away, incase somebody is routinely using this option to start the program, mixing and migrating these options cause havoc (although, there is not any critical data loss).

"alternative user directory for data and plugins"